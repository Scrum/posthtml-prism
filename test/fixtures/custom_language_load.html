<pre><code class="language-php">
    $app->post('framework/{id}', function($framework) {        
        $this->dispatch(new Energy($framework));
    });
</code></pre>