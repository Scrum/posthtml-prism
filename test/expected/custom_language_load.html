<pre><code class="language-php">
    <span class="token variable">$app</span><span class="token operator">-></span><span class="token function">post</span><span class="token punctuation">(</span><span class="token string single-quoted-string">'framework/{id}'</span><span class="token punctuation">,</span> <span class="token keyword">function</span><span class="token punctuation">(</span><span class="token variable">$framework</span><span class="token punctuation">)</span> <span class="token punctuation">{</span>
        <span class="token variable">$this</span><span class="token operator">-></span><span class="token function">dispatch</span><span class="token punctuation">(</span><span class="token keyword">new</span> <span class="token class-name">Energy</span><span class="token punctuation">(</span><span class="token variable">$framework</span><span class="token punctuation">)</span><span class="token punctuation">)</span><span class="token punctuation">;</span>
    <span class="token punctuation">}</span><span class="token punctuation">)</span><span class="token punctuation">;</span>
</code></pre>
